### PR TITLE
HydraBuild (WIP - v159 BETA)

### PR DESCRIPTION
**Pull request by CDAGaming**
Features:

- Adds Back Live Pokemon and Item Stats
- Adds New Build.bat to detect and build Automatically based on VS Version ( From @Furtif )
- Adds a new Dark Mode & Improves Theme Selection Logic
- Whether Console was Shown or Hidden as well as Other Settings are now Saved
- New GUI Console Colors to Improve Readability ( From @morbosc )
- Fixes Issues with Enable and Disable Hub (No More Message Dialog!)
- Fixes Browser Rendering Issues
- Fixes Some Translation Typos

Bugs:

- Theme and Scheme Settings do not seem to Be Saving Properly, and Reset on every new Launch

### CODE REVIEW

File: build.bat [hunk 1]
```diff
@@ -1,23 +1,13 @@
...
 :VS2017_Community
 echo Building under Visual Studio 2017 Community as %processor_architecture%
-if "%processor_architecture%"=="AMD64" (
 nuget.exe restore "NecroBot-Private for Pokemon GO.sln"
-"C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\MSBuild\15.0\Bin\MSBuild.exe" "NecroBot-Private for Pokemon GO.sln" /property:Configuration=Release /property:Platform=x64
-) ELSE (
-nuget.exe restore "NecroBot-Private for Pokemon GO.sln"
-"C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\MSBuild\15.0\Bin\MSBuild.exe" "NecroBot-Private for Pokemon GO.sln" /property:Configuration=Release /property:Platform=x86
-)
+"C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\MSBuild\15.0\Bin\MSBuild.exe" "NecroBot-Private for Pokemon GO.sln" /property:Configuration=Release /property:Platform="Any CPU"
```
Comment (Furtif) [inline build.bat]: goto end

File: PoGo.Necrobot.Window/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -51,6 +51,6 @@
 // [assembly: AssemblyVersion("1.0.0.134*")]
```
Comment (Lord-Haji) [inline PoGo.Necrobot.Window/Properties/AssemblyInfo.cs]: Just Remove this too

File: PoGo.Necrobot.Window/Properties/AssemblyInfo.cs [hunk 3]
```diff
@@ -5,12 +5,12 @@
...
 // set of attributes. Change these attribute values to modify the information
 // associated with an assembly.
-[assembly: AssemblyTitle("PoGo.Necrobot.Window")]
+[assembly: AssemblyTitle("NecroBot-Private for Pokémon GO")]
 [assembly: AssemblyDescription("")]
 [assembly: AssemblyConfiguration("")]
 [assembly: AssemblyCompany("")]
 [assembly: AssemblyProduct("PoGo.Necrobot.Window")]
-[assembly: AssemblyCopyright("Copyright ©  2016")]
+[assembly: AssemblyCopyright("Copyright NecroBot-Private Team © 2017")]
```
Comment (Lord-Haji) [inline PoGo.Necrobot.Window/Properties/AssemblyInfo.cs]: Just Remove this too

**Comment by CDAGaming** [comment]
Build will be Fixed today, waiting on a Dependency Update

**Comment by jjskuld** [comment]
@CDAGaming Going forward mega pulls like this should be avoided since it's too difficult to sift through all these changed lines.  Please try to break big pulls like this into small feature pulls.

**Comment by jjskuld** [comment]
@CDAGaming Also, you should revert the change to AssemblyInfo.cs since the version is automatically changed by the build machine and shouldn't be manually changed.  You'll get a conflict if another release is made before your pull is merged in.

**Comment by jjskuld** [comment]
The FodyWeaver changes broke the build
```
MSBUILD : error : Fody: Found no debug symbols. [c:\projects\necrobot2\PoGo.NecroBot.CLI\PoGo.NecroBot.CLI.csproj]
```

**Comment by hardknoxni69a** [comment]
I Couldnt Agree More. Smaller Individual Pull Requests Per Feature. And Things Will Get Done Much Quicker Massive Code Changes Like This 46K Lines Of Code Is Gonna Sit Forever. Where As You Do One Feature Make A PR And Then It Gets Merged But This Is To Much To Merge At Once Safely. My 2Cents

**Comment by CDAGaming** [comment]
@jjskuld That is not because of FodeWeavers, it is from a bug on FODY's end, and i and several contributors have been working with them to fix this, and it has been fixed today, await the updated commit

**Comment by CDAGaming** [comment]
The 40k+ Changes were added mainly because of the tab images and theme logic, as if you went through the actual changes, it's nowhere near that massive amount

Commit that had Fixed Appveyor/MSBuild Errors: https://github.com/Fody/Costura/commit/0d6c57c079f80c364dd304582f069b46ce81ebc2

**Comment by CDAGaming** [comment]
ATTENTION: This PR Will be closed soon to split this up into several feature and fixes prs for official merge, this will happen after fixes tomorrow(Hopefully)

**Comment by CDAGaming** [comment]
PR Will be split up today

**Pull request closed without merging**